app/assets/stylesheets/editions.scss

```diff
@@ -14,3 +14,10 @@
     margin-top: govuk-spacing(3)
   }
 }
+
+.edition__edit_assignee {
+  p {
+    padding-left: govuk-spacing(2);
+    font-size: 20px;
```

Comment:
We should avoid using explicit pixel sizes for font-size. Can we use the [design system type scale](https://design-system.service.gov.uk/styles/type-scale/) instead? Also I don't think 20px is a size we generally use.

Reply:
Ah, very good catch. I'll take a look at this.

Reply:
Actually this comment is probably rendered irrelevant by the one below (in the `edit_assignee` partial) if that is applicable.